### PR TITLE
Remove copy paste error, small change to README

### PR DESCRIPTION
**Pull request by pritchyspritch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,27 +24,29 @@ Within the AWS account you wish to deploy to.
...
+- `terraform init; terraform plan`
+
+*AWS Vault: `aws-vault exec <profile> -- terraform init; terraform plan`*
 
 - If the plan looks good then run:
-terraform apply
+`terraform apply`
 
-- To obtain the public ip address run aws-vault exec <profile> -- aws ec2 describe-instances | grep 'GroupName\|PublicIp' 
+- To obtain the public ip address run `aws-vault exec <profile> -- aws ec2 describe-instances --region eu-west-2`
```
Comment (akinnane) [inline README.md]: Could this be an `output` from the terraform?

**Review by sernoval — DISMISSED** [review]
LGTM

File: terraform/main.tf [hunk 2]
```diff
@@ -107,7 +107,7 @@ data "template_file" "kali_userdata" {
     hostname        = "kali-pentest-01"
     ssh-pub-key-1   = "${local.ssh-pub-key-1}"
     ssh-pub-key-2   = "${local.ssh-pub-key-2}"
```
Comment (akinnane) [inline terraform/main.tf]: Could this be a list instead? 
```suggestion
    ssh-keys  = ["${local.ssh-pub-key-1}", "${local.ssh-pub-key-2}", "${local.ssh-pub-key-3}"]
```

File: terraform/cloudinit/kali-instance.yaml [hunk 3]
```diff
@@ -13,6 +13,7 @@ users:
     ssh_authorized_keys:
       - ${ssh-pub-key-1}
       - ${ssh-pub-key-2}
+      - ${ssh-pub-key-3}
```
Comment (akinnane) [inline terraform/cloudinit/kali-instance.yaml]: https://www.terraform.io/docs/configuration/expressions.html#directives
Loops in templates
```suggestion
    ssh_authorized_keys:
    %{ for key in ssh-keys ~}
      - ${key}
    %{ endfor ~}
```

File: terraform/main.tf [hunk 4]
```diff
@@ -107,7 +107,7 @@ data "template_file" "kali_userdata" {
...
-    ssh-pub-key-2   = "${local.ssh-pub-key-3}"
+    ssh-pub-key-3   = "${local.ssh-pub-key-3}"
```
Comment (akinnane) [inline terraform/main.tf]:
```suggestion
```

**Review by akinnane — APPROVED** [review]
LGTM

**Pull request merged**